index.html

```diff
@@ -0,0 +1,99 @@
+<html>
+    <head>
+        <title>Guillain</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    <body class="container-fluid">
+        <div style="position: fixed" class="mt-4">
+            <nav id="navbar" class="nav-fill">
+                <header class="h3 text-secondary">Lorem Ipsum!</header>
+                <ul class="nav flex-column float-left">
+                    <li class="nav-item">
+                        <a href="#What_is_Lorem_Ipsum" class="nav-link">What is Lorem Ipsum?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Where_does_it_come_from" class="nav-link">Where does it come from?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Why_do_we_use_it" class="nav-link">Why do we use it?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#Where_can_I_get_some" class="nav-link">Where can I get some?</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#standard_Lorem_Ipsum" class="nav-link">The standard Lorem Ipsum passage</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <main class="mt-4" style="width: 78%; position: absolute; right: 0;" id="main-doc">
+            <section class="main-section" id="What_is_Lorem_Ipsum">
+                <header class="h2" >What is Lorem Ipsum?</header>
+                <code>What is Lorem Ipsum?</code>
+                <p>
+                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                </p>
+                <ul>
+                    <li>At vero et odio </li>
+                    <li>eos et</li>
+                    <li>accusamus</li>
+                    <li>iusto</li>
+                    <li>dignissimos</li>
+                    <li>survived</li>
+                </ul>
+                <p>
+                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                </p>
+            </section>
+            <section class="main-section" id="Where_does_it_come_from">
```

Comment:
You are using an unused CSS selector `main-section`

Reply:
The same issue as above! you should be polite and explain your reasoning as to why the change is needed!